grc/tests/CMakeLists.txt

```diff
@@ -0,0 +1,40 @@
+# Copyright 2011-212 Free Software Foundation, Inc.
```

Comment:
Please fix the dates here to just be 2020

Reply:
Done